translate/tests/system.py

```diff
@@ -56,7 +56,7 @@ def test_translate(self):
         values = ['hvala ti', 'dankon',
                   'Me llamo Jeff', 'My name is Jeff']
         translations = Config.CLIENT.translate(
-            values, target_language='de', model=translate.NMT)
+            values, target_language='de', model='nmt')
```

Comment:
It will work, but it is not the format we want to advertise. Specific request from the API team.